packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -446,6 +446,32 @@ const ArrayMixin = Mixin.create(Enumerable, {
     which you can use to override the method names that will be invoked on the
     target.
 
+    Example:
```

Comment:
YUIDoc supports an `@example` annotation with a description https://yui.github.io/yuidoc/syntax/index.html. Could you update for that?

Reply:
I've moved the example to below an `@example` tag, let me know if the position is ok.